app/src/main/java/com/example/dynamicdiet/dao/IPersonDao.kt

```diff
@@ -1,6 +1,6 @@
 //person DAO fill out more with when connection to data becomes a thing
 
-interface ICountryDao {
+interface IPersonDao { // Changed it from ICountryDao to IDietDao.
```

Comment:
Conversation in code.  Also, the conversation is incorrect (IPersonDao/IDietDao).